weltyt.py

```diff
@@ -0,0 +1 @@
+print("Welcome to tyton")
```

Comment:
Always end scripts with newline.
See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline if you want to know why.

Reply:
Thank you for advices.